### PR TITLE
Remove default template dependency on client side CSS

### PR DESCRIPTION
**Pull request by TienSFU25**
See Issue #8126

Adds these files under /shared
- normalizer.css
- error.css
- extracted.css (for subscribers.css and private.css)

Also makes these files available as public static content

### CODE REVIEW

File: core/server/apps/private-blogging/lib/views/private.hbs [hunk 1]
```diff
@@ -15,7 +15,8 @@
         <link rel="shortcut icon" href="{{asset "favicon.ico"}}">
         <meta http-equiv="cleartype" content="on">
 
-        <link rel="stylesheet" href="{{asset "ghost.css" ghost="true" minifyInProduction="true"}}" />
+        <link rel="stylesheet" href="/normalizer.css"/>
+        <link rel="stylesheet" href="/extracted.css"/>
```
Comment (AileenCGN) [inline core/server/apps/private-blogging/lib/views/private.hbs]: `extracted.css` is not served.

File: core/server/apps/subscribers/lib/views/subscribe.hbs [hunk 2]
```diff
@@ -15,7 +15,8 @@
     <link rel="shortcut icon" href="{{asset "favicon.ico"}}">
     <meta http-equiv="cleartype" content="on">
 
-    <link rel="stylesheet" href="{{asset "ghost.css" ghost="true" minifyInProduction="true"}}" />
+    <link rel="stylesheet" href="/normalizer.css"/>
+    <link rel="stylesheet" href="/extracted.css"/>
```
Comment (AileenCGN) [inline core/server/apps/subscribers/lib/views/subscribe.hbs]: Same as above, `extracted.css` doesn't exist.

File: core/server/blog/app.js [hunk 3]
```diff
@@ -48,6 +47,12 @@ module.exports = function setupBlogApp() {
     blogApp.use(serveSharedFile('sitemap.xsl', 'text/xsl', utils.ONE_DAY_S));
     // Serve robots.txt if not found in theme
     blogApp.use(serveSharedFile('robots.txt', 'text/plain', utils.ONE_HOUR_S));
+
+    // Serve stylesheets for default templates
+    blogApp.use(serveSharedFile('normalizer.css', 'text/css', utils.ONE_HOUR_S));
+    blogApp.use(serveSharedFile('error.css', 'text/css', utils.ONE_HOUR_S));
+    blogApp.use(serveSharedFile('subscribers.css', 'text/css', utils.ONE_HOUR_S));
```
Comment (AileenCGN) [inline core/server/blog/app.js]: Instead of using three different css files, you could simplify it and just serve all the styles from one file instead.

**Review by aileen — CHANGES_REQUESTED** [review]
`private.hbs` and `subscribe.hbs` are requesting a css file that doesn't get served. This would need to be corrected, to make it work.

`error.hbs` is looking good to me.

File: core/server/blog/app.js (hunk 3 above)
Comment (TienSFU25) [inline core/server/blog/app.js]: okay. what should this file be called? `shared.css` sounds too generic.

**Comment by TienSFU25** [comment]
Also do I need to add unit tests for adding the shared file?

File: core/server/blog/app.js (hunk 3 above)
Comment (AileenCGN) [inline core/server/blog/app.js]: I would probably just go for `ghost.css`, the way it was before?

**Comment by TienSFU25** [comment]
why is the core/client stuff in here? I don't understand

**Comment by kevinansfield** [comment]
@TienSFU25 the `core/client` changes are there because you staged the `core/client` submodule before committing.

To remove them now that you've committed you would need to go through the following steps:
1. `cd core/client`
2. find the ref that `master` currently points to (see image below)
3. `git checkout {{ref}}` (in this case `git checkout a8e58bf`)
4. `cd ../../`
5. `git add core/client`
6. `git commit -m "reset core/client submodule"` (or if you want to change the existing commit you could do `git commit --amend --no-edit`)

screen shot 2017-03-28 at 15 10 40

**Comment by ErisDS** [comment]
This was merged in #8234, thanks @TienSFU25 for your hard work!

I know you've had trouble getting in touch with us to ask questions - we are active in slack but most of us are in UK/EUR timezones. The #core-dev channel is specifically for asking contributing related questions & we've worked on overhauling our tools and docs to start making it easier. Let us know if there's anything else we can do.

**Pull request closed without merging**